schemas/hygrothermalData.json

```diff
@@ -1,37 +1,183 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://www.buildingenvelopedata.org/schemas/v1/hygrothermalData.json",
+  "$id": "https://github.com/building-envelope-data/api/tree/develop/schemas/hygrothermalData.json",
   "title": "hygrothermal data",
   "description": "Hygrothermal data which can be used for hygrothermal simulations. Humidity needs to be handled properly to create sustainable buildings with high comfort.",
   "$defs": {
     "data": {
-      "title": "The hygrothermal data of a component.",
+      "title": "Hygrothermal data of a component",
+      "description": "A schema including all hygrothermal properties which are currently relevant for Architecture, Engineering and Construction.",
       "type": "object",
       "properties": {
-        "WUFI": {
-          "title": "Hgrothermal data from the database of WUFI.de.",
-          "type": "object",
-          "properties": {
-            "id": {
-              "$ref": "identifier.json#/$defs/decentral",
-              "description": "The id represents the hygrothermal data set."
-            },
-            "description": {
-              "type": "string",
-              "description": "An explanation with more details about the hygrothermal data in natural language."
-            },
-            "dataFormat": {
-              "$ref": "common.json#/$defs/webAddress",
-              "description": "The web address provides a definition of the format of the 'xmlData'."
-            },
-            "xmlData": {
-              "$ref": "string.json#/$defs/extensibleMarkupLanguageDocument",
-              "description": "Hygrothermal data in Extensible Markup Language (XML) format in the exact data format specified by the property 'formatting'."
+        "thermalConductivityDry": {
+          "$ref": "number.json#/$defs/wattPerMeterKelvin"
```

Comment:
Please add a description to each property which explains the property to a software developer who has no experience with hygrothermal data.